metron-deployment/README.md

```diff
@@ -15,178 +15,132 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
-This will set up
-- Metron Parsers
-- Enrichment
-- Indexing
-- GeoIP data
-- Optional Elasticsearch
-- Optional Kibana
-
-### Prerequisites
+This project contains a variety of tools for deploying Apache Metron.  Please refer to the following sections for more information on the best way to deploy Metron in your environment.
+
+ * [How do I deploy Metron on a single VM?](#how-do-i-deploy-metron-on-a-single-vm)
+ * [How do I deploy Metron on a large cluster with Ambari?](#how-do-i-deploy-metron-on-a-large-cluster-with-ambari)
+ * [How do I build RPM packages?](#how-do-i-build-rpm-packages)
+ * [How do I build DEB packages?](#how-do-i-build-deb-packages)
+ * [How do I deploy Metron within AWS?](#how-do-i-deploy-metron-within-aws)
+
+## How do I deploy Metron on a single VM?
+
+#### Who is this for?
+
+If you are new to Metron and just want to explore the functionality that it offers, this is good place to start.  If you are a developer building on Metron, then this is also a great way to test out your changes.
+
+#### What does this do?
+
+This option will deploy Metron on a virtual machine running on your computer.  This is often referred to as the "Full Dev" environment.  
+
+#### How do I do it?
+Follow these [instructions to deploy Metron on a single VM](vagrant/full-dev-platform/README.md).
+
+#### Where can I get more information?
+* [Full Dev Platform](vagrant/full-dev-platform)
+
+## How do I deploy Metron on a large cluster with Ambari?
+
+#### Who is this for?
+
+If you want to see how Metron can really scale by deploying it on your own hardware, or even in your own cloud-based environment, this is the best option for you.
+
+#### What does this do?
+
+This creates a Management Pack (MPack) extension for [Apache Ambari](https://ambari.apache.org/) that simplifies the provisioning, managing and monitoring of Metron on large clusters.  This allows you to easily install Metron on a large cluster using a simple, guided process.  This also allows you to monitor cluster health and even secure your cluster with kerberos.
+
+#### How do I do it?
+
+##### Prerequisites
+
 - A cluster managed by Ambari 2.4.2+
-- Metron RPMs available on the cluster in the /localrepo directory.  See [RPMs](#rpms) for further information.
-- [Node.js](https://nodejs.org/en/download/package-manager/) repository installed on the Management UI host
+- Installable Metron packages (either RPMs or DEBs) available on the cluster, by default, in the `/localrepo` directory.  
+- A [Node.js](https://nodejs.org/en/download/package-manager/) repository installed on the host running the Management and Alarm UI.
+
+##### How do I build the Mpack?
+
+```
+cd metron-deployment
+mvn clean package -Pmpack
+```
 
-### Building Management Pack
-From `metron-deployment` run
+This results in the Mpack being produced at the following location.
 ```
-mvn clean package
+metron-deployment/packaging/ambari/metron-mpack/target/metron_mpack-0.4.2.0.tar.gz
```

Comment:
Hi @nickwallen , would it be better to note the version like `metron_mpack-x.x.x.x.tar.gz` instead of 0.4.2.0 so we need not have to update the README over releases?

Reply:
Yes, you are right @anandsubbu .  Thanks